packages/dataprotector-consumer/README.md

```diff
@@ -0,0 +1,38 @@
+<p align="center">
+  <a href="https://iex.ec/" rel="noopener" target="_blank"><img width="150" src="./logo-iexec.png" alt="iExec logo"/></a>
+</p>
+
+<h1 align="center">DataProtector Consumer</h1>
+
+This package helps nodejs iExec dapps to consume protected data created with `@iexec/dataprotector`
+
+<div align="center">
+
+[![npm](https://img.shields.io/npm/v/@iexec/dataprotector-consumer)](https://www.npmjs.com/package/@iexec/dataprotector-consumer) [![license](https://img.shields.io/badge/license-Apache%202-blue)](/LICENSE)
+
+</div>
+
+## Installation
+
+**npm:**
+
+```sh
+npm install @iexec/dataprotector-consumer
+```
+
+**yarn:**
+
+```sh
+yarn add @iexec/dataprotector-consumer
+```
+
+## Get started
+
+In your nodejs iExec dapp
```

Comment:
what about python compatibility ?

Reply:
this is a js package but the logic can be replicated in python with [borsh-construct-py](https://github.com/near/borsh-construct-py)

we can add this to the documentation

```py
from borsh_construct import Bool, String, F64, U128

# bool schema
serializedBool = Bool.build(True)
print("serializedBool", serializedBool)
deserializedBool = Bool.parse(serializedBool)
print("deserializedBool", deserializedBool)

# string schema
serializedString = String.build("foo bar")
print("serializedString", serializedString)
deserializedString = String.parse(serializedString)
print("deserializedString", deserializedString)

# f64 schema
serializedF64 = F64.build(1.00000000001)
print("serializedF64", serializedF64)
deserializedF64 = F64.parse(serializedF64)
print("deserializedF64", deserializedF64)

# i128 schema
serializedU128 = U128.build(1234567890)
print("serializedU128", serializedU128)
deserializedU128 = U128.parse(serializedU128)
print("deserializedU128", deserializedU128)
```